app/src/androidTest/java/com/github/sdpcoachme/data/UserInfoSamples.kt

```diff
@@ -0,0 +1,54 @@
+package com.github.sdpcoachme.data
+
+class UserInfoSamples {
```

Comment:
Nice refactoring idea! Now, we can mitigate unnecessary code duplications for creating users